delivery.yaml

```diff
@@ -7,6 +7,8 @@ pipeline:
     - /go/pkg/mod       # pkg cache for Go modules
     - ~/.cache/go-build # Go build cache
   type: script
+  env:
+    GOFLAGS: "-mod=readonly"
```

Comment:
you should just move forward to Go 1.13 and commit the change of go.mod

Reply:
I did that, but I'd still like the build to just fail instead of silently doing things in the background. No idea why this isn't the default.